### PR TITLE
Add disableAutoQuoting option to stop quoting of string object properties

### PR DESCRIPTION
**Pull request by theRobinator**
See https://github.com/angular/tsickle/issues/683 for motivation.

### CODE REVIEW

File: src/main.ts [hunk 1]
```diff
@@ -66,6 +70,8 @@ function loadSettingsFromArgs(args: string[]): {settings: Settings, tscArgs: str
       case 'verbose':
         settings.verbose = true;
         break;
+      case 'disableAutoQuoting':
+        settings.disableAutoQuoting = true;
```
Comment (mprobst) [inline src/main.ts]: don't forget the break!

Comment (theRobinator) [reply src/main.ts]: Whoops! Good catch. Done.

**Comment by mprobst** [comment]
@theRobinator could you rebase the CL on `origin/master` and force push? That should get rid of the commit by me in the history and make the CLA bot happy.

```
git rebase origin/master
git push -f
```

**Comment by theRobinator** [comment]
Will do.

**Comment by theRobinator** [comment]
@mprobst done!

**Comment by mprobst** [comment]
merged.

**Comment by alexeagle** [comment]
Note, we now think the option was reversed - autoQuoting should be opt-in, not opt-out (at least until it works almost all the time)

https://github.com/angular/tsickle/pull/895 will switch the logic to `enableAutoQuoting` instead.

**Pull request merged**